index.html

```diff
@@ -0,0 +1,159 @@
+<html>
+<head>
+	<title>Cognits</title>
+	<link rel="stylesheet" type="text/css" href="styles/bootstrap.min.css">
+	<link rel="stylesheet" type="text/css" href="styles/bootstrap-theme.min.css">
+	 <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+	<link rel="stylesheet" type="text/css" href="styles/styles.css">
+</head>
+<body>
+<nav class="navbar navbar-inverse navbar-fixed-top ">
+	<div class="container-fluid">
+		<ul>
+			<li><img src="img/cognits_logo_wht.png" class="img-responsive Cognist"></li>
+		</ul>
+	</div>
+</nav>
+	<div class = "container-full" id= "fondo" >
+		<div class="col-md-3"> </div>
+		<div class="col-md-6">
+		<img src="img/title_intro.png" class="img-responsive" id="posicionImg">
+		</div>
+		<div class="col-md-3"></div>
+	</div>
+<div class="container-full  text-center">
+	<div class = 'row WouldLike '>
+		<h1 class="OurApprenticesTitle">Our apprentices would like to work with you</h1>
+	</div>
+	<div class = "row WouldLike">
+
+		<div class="col-md-3">
+			<p class="ParagraphWouldLike">"besides the technical learning, Cognits has also focused on teaching us to be better person , teaching us grat moral values and practices."</p>
+			<p>Frank Sam Hu,</p>
+		</div>
+		<div class="col-md-4">
+			<p class="ParagraphWouldLike">"Thank to Cognits Ihave obtained personal and professional victories, I've dreamed more ambitious than before, I have dream and hoal bigger than what one day I thought I would have."</p><p>Jose Alejandro Tamayo</p>
+		</div>
+		<div class="col-md-3">
+			<p class="ParagraphWouldLike">"Cognits has taugh me that the impossible only exists in the mind" </p>
+			<p>Kevin Herrera,</p>
+		</div>
+		<div class="col-md-1"> <img src="img/back_quotes-03.png" class="image"></div>
+	</div>
```

Comment:
Aqui existe un problema con el tamaño de la imagen `img/back_quotes-03.png`, y las columnas. Primero, las columnas suman a **11**, y en Bootstrap siempre deben sumar a **12**.

Sugiero que la imagen no la metan en columna, pues creara muchos problemas en el responsive. En vez, busquen una manera de tener la imagen atras. Hay varias formas de hacer esto, como: 
- [usar el z-index](http://philipwalton.com/articles/what-no-one-told-you-about-z-index/)
- [posicionamiento de CSS](http://www.w3schools.com/css/css_positioning.asp)
- Buscar alguna solucion en [CSS-Tricks](https://css-tricks.com/).

No se cual sera la mejor solucion, sera de probar y ver que si funciona y mira re bien de forma responsive.